### ISSUE TITLE
Reins of kor'kron annihilator BUGGED

### ISSUE THREAD

**Issue by Iqdoc** [gamefreedomgit/Maelstrom#3450]
[//]: # Guild Perk Bugged.
[//]: # [(http://wowhead.com/)](https://www.wowhead.com/item=67107/reins-of-the-korkron-annihilator)

You get exalted and you cant buy it. Cause its not in the buy list. You can buy the Dark Phoenix on the same vendor but there is no Reins of kor'kron annihilator to buy. In the guild Rewards tab says with Red : Requires Guild Achievment Guild Level 25. But guild is lvl 25 since i can buy the Reins of the Dark Phoenix

**Comment by Iqdoc** [comment]
Nothing to say. You get exalted and you cant buy it. Cause its not in the buy list. You can buy the `Dark Phoenix` on the same vendor but there is no Reins of kor'kron annihilator to buy.

**Comment by hristoforyordanov92** [comment]
**Description:**
I think I've figured why the [Reins of the Kor'kron Annihilator](https://cata-twinhead.twinstar.cz/?item=67107) is impossible to buy, even when your guild is level 25 and you're exalted with your guild.
I am an undead player, member of the guild "Farmers Market" and they have been level 25 for a while.
Got my Exalted with them last night, but when checking the vendor, the mount is not there. In the Rewards tab of the guild panel, it says that the mount is locked and it requires [Guild Level 25 (Horde)](https://cata-twinhead.twinstar.cz/?achievement=5492) achievement. And in the Guild tab of the Achievements panel, the achievement in question is not completed. However, I scrolled up and I saw that this achievement [Guild Level 25 (Alliance)](https://cata-twinhead.twinstar.cz/?achievement=4912) is completed. And of course - the guild master is an alliance player and that makes sense why the horde achievement is not completed.

**How to reproduce:**
1. Make a guild with an alliance character.
2. Get the guild to level 25.
3. Join the guild with a horde character.
4. Get the horde character to an exalted status with the guild.
5. See that the mount is not available in the guild vendor.

**How it should work:**
Ideally, each achievement from the Guild Level chain should trigger the corresponding (by level) achievement for the other faction.

**Database links:**
Links are in description above.
